packages/alfa-rules/src/sia-r87/rule.ts

```diff
@@ -73,20 +73,23 @@ export default Rule.Atomic.of<Page, Document, Question>({
           "first-tabbable-reference-is-main",
           "boolean",
           target,
+          target,
```

Comment:
This question (and others in this rule) should know be able to be asked about the first tabbable rather than the document. But that is maybe a topic for another PR 🤔

Reply:
Agreed! That's another, and slightly more involved, breaking change so let's deal with that in another pull request.